src/test/java/tests/ExcludeLanguagesTest.java

```diff
@@ -0,0 +1,85 @@
+package tests;
+
+import Page_Objects.TestBase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class ExcludeLanguagesTest {
+
+    private ChromeDriver driver;
+    private TestBase testBase;
+
+    String portugueseLang = "Portuguese";
+    String englishLang = "English";
+    String portSong = "os-quatro-e-meia-ola-solidao";
+    String engSong = "bob-marley-the-wailers-three-little-birds";
+    String portEngSong = "nelly-furtado-forca";
+
+    @BeforeEach
+    public void setUp() {
+        driver = new ChromeDriver();
+        testBase = new TestBase(driver);
+        testBase.setUp();
+    }
+
+    @Test
+    public void excludingSongLanguages() {
+
+        // Step 1: Select Advanced setup
+        testBase.getLandingPage().enterTheGame();
+        testBase.getInputSelectionPage().selectAdvancedSetup();
+        testBase.getAdvancedConnectionPage().goToMainMenu();
+        testBase.getMainMenuPage().goToSongList();
+
+        // Step 2: Exclude Portuguese language
+        testBase.getSongLanguagesPage().unselectLanguage(portugueseLang);
+        testBase.getSongLanguagesPage().ensureSongLanguageIsSelected(englishLang);
+        testBase.getSongLanguagesPage().continueAndGoToSongList();
+        Assertions.assertFalse(testBase.getSongListPage().isSongOnTheList(portSong));

```

Comment:
The song could not be on the list because e.g. the list isn't loaded yet. I would move this line down and start with `Assertions.assertTrue` assertions just to be safe - if they pass, it means some songs are actually visible

Reply:
Thanks, done :)